runtime/pom.xml

```diff
@@ -44,36 +44,43 @@
     <finalName>runtime</finalName>
     <plugins>
       <plugin>
-        <groupId>org.wildfly.swarm</groupId>
-        <artifactId>wildfly-swarm-plugin</artifactId>
-        <version>${wildfly.swarm.version}</version>
-        <configuration>
-          <mainClass>com.redhat.ipaas.runtime.Main</mainClass>
-          <jvmArguments>
-            <jvmArgument>-server</jvmArgument>
-            <jvmArgument>-Xms256m</jvmArgument>
-            <jvmArgument>-Xmx512m</jvmArgument>
-          </jvmArguments>
```

Comment:
It sure does matter even during development so it limits memory consumption. Plus it's a great placeholder to set these limits. It's much easier to debug a JVM to run out of memory then it is for OpenShift to blow itself up. Speaking from experience.